google/cloud/aiplatform/training_jobs.py

```diff
@@ -339,6 +339,190 @@ def package_and_copy_to_gcs(
         return self.package_and_copy(copy_method=copy_method)
 
 
+class _MachineSpec(NamedTuple):
+    """Specification container for Machine specs used for distributed training.
+
+    Usage:
+
+    spec = _MachineSpec(
+                replica_count=10,
+                machine_type='n1-standard-2',
+                accelerator_count=2,
+                accelerator_type='NVIDIA_TESLA_K80')
+
+    Note that container and python package specs are not stored with this spec.
+    """
+
+    replica_count: int = 0
+    machine_type: str = "n1-standard-2"
```

Comment:
`n1-standard-2` is supported in Deploy but not for Training. 
`n1-standard-4` is currently the lowest common denominator.

Reply:
- [Machine types for prediction](https://cloud.devsite.corp.google.com/ai-platform-unified/docs/predictions/machine-types#machine_type_comparison)
- [Machine types for training](https://cloud.devsite.corp.google.com/ai-platform-unified/docs/training/configure-compute?db=piac#specifying_machine_types)

Reply:
Thanks! That's good to know. I was working off the machine_resource.MachineSpec proto comments.

Reply:
Done.